src/ir/local-graph.h

```diff
@@ -74,7 +74,8 @@ struct LocalGraph {
   bool equivalent(LocalGet* a, LocalGet* b);
 
   // Optional: compute the influence graphs between sets and gets (useful for
-  // algorithms that propagate changes).
+  // algorithms that propagate changes). Set influences are the gets that can
+  // read from it; get influences are the sets that can (directly) read from it.
```

Comment:
```suggestion
  // read from it; get influences are the sets from which the get can (directly) read.
```

Reply:
IIUC what you wrote, that is actually `getSets`, not `getInfluences`. Example:

```js
x = 5;       // line 0
foo(x);      // line 1
y = x + 20;  // line 2
```
* `getSets` of the `x` on line 1 is the set of `x` on line 0.
* `setInfluences` of the set on line 0 are the gets of `x` on lines 1 and 2.
* `getInfluences` of the `x` on line 2 is the set of `y` on line 2.

`getSets` is the basic property of "what value is being read here?" We go backwards to find the right sets.

*Influences are used for forward propagation (hence the comment says "useful for algorithms that propagate changes"). Imagine that a set is given a new value, then `setInfluences` is the gets that read it. And, from each of those gets, we want to know which other sets may be influenced by it in turn, to keep propagating forward.

Reply:
I see. I was confused by the explanation that sets were doing reads and thought that was surely a typo. Perhaps instead of talking about sets doing reads, we can describe the sets' values as using the gets.

Reply:
Perhaps. Maybe "propagates" can also make more sense rather than "influences", as influence isn't as clearly directional. I'm not sure what's best here, but let's keep thinking about it.